en/assembly/quick_start_pixhawk4.md

```diff
@@ -155,13 +163,13 @@ General configuration information is covered in: [Autopilot Configuration](../co
 
 QuadPlane specific configuration is covered here: [QuadPlane VTOL Configuration](../config_vtol/vtol_quad_configuration.md)
 
-<!-- what about config of other vtol types and plane. Do the instructions in these ones above apply for tailsitters etc? -->
+<!-- what about config of other vtol types and plane. Do the instructions in these ones above apply for tailsitters etc? --> 

```

Comment:
Not really. No such instructions exist. I have tried to get them, but no traction.